src/main/java/ezvcard/io/json/JCardRawWriter.java

```diff
@@ -333,7 +337,12 @@ private void init() throws IOException {
 
 		if (wrapInArray) {
 			generator.writeStartArray();
-			indent(0);
+		}
+
+		if (isIndent()) {
```

Comment:
Can you change `isIndent()` to `isPrettyPrint()` and `setIndent()` to `setPrettyPrint()`?